bitnami/spring-cloud-dataflow/README.md

```diff
@@ -374,17 +375,16 @@ Only one messaging layer can be used at a given time.
 
 ### Using an external database
 
-Sometimes you may want to have Spring Cloud components connect to an external database rather than installing one inside your cluster, e.g. to use a managed database service, or use run a single database server for all your applications. To do this, the chart allows you to specify credentials for an external database under the [`externalDatabase` parameter](#database-parameters). You should also disable the MariaDB installation with the `mariadb.enabled` option. For example with the following parameters:
+Sometimes you may want to have Spring Cloud components connect to an external database rather than installing one inside your cluster, e.g. to use a managed database service, or use run a single database server for all your applications. To do this, the chart allows you to specify credentials for an external database under the [`externalDatabase` parameter](#database-parameters). You should also disable the MariaDB installation with the `mariadb.enabled` option. For example, to configure the dataflow server to connect to MS SQL Server:
 
 ```console
 mariadb.enabled=false
-externalDatabase.host=myexternalhost
-externalDatabase.port=3306
+externalDatabase.url=jdbc:sqlserver://mssql-server:1433
 externalDatabase.password=mypassword
 externalDatabase.dataflow.user=mydataflowuser
-externalDatabase.dataflow.database=mydataflowdatabase
 externalDatabase.dataflow.user=myskipperuser
-externalDatabase.dataflow.database=myskipperdatabase
+externalDatabase.hibernateDialect=org.hibernate.dialect.SQLServer2012Dialect
+
```

Comment:
I would leave the previous external database definition as we are not bumping a major and most users will still be using the old approach. We could add a note or a new console example using the `externalDatabase.url` parameter instead, what do you think?

Reply:
OK, I left the original configuration and then I added an additional section to demonstrate how to use MS SQL Server instead